src/worker.py

```
@@ -1539,10 +1548,89 @@ async def api_get_activity(act_id: str, req, env):
         "enrollment":  {
             "role":   enrollment.role,
             "status": enrollment.status,
+            "id":     enrollment.id,
         } if enrollment else None,
     })
 
 
+async def api_generate_certificate(request, env, enrollment_id):
```

Ruff (0.15.15)

[warning] Too many return statements (7 > 6)

(PLR0911)